app/src/stylesheets/buyButton.scss

```diff
@@ -0,0 +1,21 @@
+.buy-btn {
```

Comment:
same here with hungarian, this is more as a reminder rather than bringing it up twice for when the name gets changed